frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -82,6 +84,8 @@ pods:
       broker:
         cpus: {{BROKER_CPUS}}
         memory: {{BROKER_MEM}}
+        shm-size: 256
```

Comment:
in 0.57.0-rc2 we removed the ability to specify shm-size if mode is share_parent since the container will inherit from the parent /dev/shm, can you remove this here?